docs/aurora/visualization/paraview.md

```diff
@@ -1 +1,110 @@
-# Paraview on Aurora
+# ParaView on Aurora
+
+The recommended way of running ParaView on Aurora is in client/server mode. This consists of running the ParaView client on your local resource and the ParaView server on the Aurora compute nodes. The ParaView client needs to first be installed on your local resource and must match the version that you run on Aurora.
+
+There may be multiple versions of ParaView installed on Aurora. To find the versions of ParaView currently available on Aurora, run the following command on a login node: 
+```
+module use /soft/modulefiles
+module avail paraview
+```
+
+Binary and source packages of the ParaView client for Linux, macOS, and Windows are available from the [ParaView Download Page](https://www.paraview.org/download/). 
+
+## Connecting to the ParaView server on Aurora
+This section describes how to launch the ParaView server on Aurora from a local ParaView client.
+
+### Start ParaView Client
+First, launch the ParaView client on your local resource. You will need to configure some server settings in the client. This initial setup should only need to be done once and can be reused each time you want to run ParaView on Aurora.
+
+### Server Configuration
+
+#### 1. Select Connect
+From the ParaView client, choose to connect to a server by either clicking on the "Connect" icon in the menu bar
+ 
+![Connect icon](images/connect-icon.png) 
+
+or selecting File->Connect from the main menu.
+
+<figure markdown>
+  ![Select connect](images/ParaviewConnectMenu.png){ width="700" }
+</figure>
+
+#### 2. Set Up Servers (first time only)
+The first time you want to run a server on Aurora and have it connect to your local ParaView client, you will need to set up a server. Once this server is set up, you can reuse it each time you run the ParaView client with the ParaView server on Aurora.
+
+Kitware, the developers of ParaView, maintain a database of server configurations that you can retrieve through the ParaView client. In the File->Connect menu, press the button named "Fetch Servers" and select AURORA@ANL. Windows users should select "windows to AURORA@ANL". Press "Import Selected".
+
+<figure markdown>
+  ![Load servers](images/ParaviewFetchServers.png){ width="700" }
+</figure>
+
+#### 3. Use ParaView
+
+After the previous step, you can now select AURORA@ANL in the File->Connect menu and press Connect.
+
+<figure markdown>
+  ![Load servers](images/ParaviewChooseServerConfig.png){ width="700" }
+</figure>
+
+At this point, a new window will pop up.
+
+<figure markdown>
```

Comment:
FYI all these HTML tags should be removed. See a4994cb8652a0588c7639431caa39a19ad7b12a7

the `{ width="700" }` etc attributes will still be respected

Reply:
fixed in [a4147dd](https://github.com/argonne-lcf/user-guides/pull/785/commits/a4147dd44ef9d06d403023b215864d68b46770b5)